src/utils/specs.py

```diff
@@ -105,6 +106,7 @@ def __str__(self):
             hbmBW:              {self.hbmBW} MB/s
             compute_partition:  {self.compute_partition}
             memory_partition:   {self.memory_partition}
+            XCCs:               {self.XCCs}
```

Comment:
```suggestion
            XCCs:               {self.XCDs}
```
Earlier (on line 72) you define this member var as `XCDs`. Is this a typo?

Reply:
Ah, yeah -- typo